### DISCUSSION TITLE
Support: Locked myself out

### DISCUSSION THREAD

**Discussion by ronnoceel** [seerr-team/seerr#598] [Q&A] [answered]
Hello. I was very excited about setting up a reverse proxy/dns for my server to give all my services cool domains on my home network. I was able to successfully do so, and set the jellyfin url to my custom domain in the Jellyseerr settings. Unfortunately, after restarting nginx, none of the domains worked anymore. No biggie, not too important to me. However, I am no longer able to login to jellyseerr because it cannot connect to Jellyfin (because the URL goes nowhere). I am wondering what the best course of action is for me. Is there a way to destroy the docker persistence and start over with no accounts?

**Comment by fallenbagel** [comment, marked as answer]
You can stop jellyseerr and edit the settings.json, and change the jellyfin url. Or just delete the config folder and start over.

**Reply by ronnoceel** [reply]
Thank you! I thought my system didn't have a config because it wasn't where I expected it to be- turns out it was at `/path/to/appdata/config/` which I never set up properly. It has since been setup and I can access Jellyseerr now!

**Reply by fallenbagel** [reply]
`/path/to/appdata/config/` is just a placeholder 😅
Maybe I should make it more clearer by putting a comment inside the dockercompose template
